index.html

```diff
@@ -117,20 +117,30 @@ <h2>API</h2>
 import * as parsel from "./parsel.js";
 self.parsel = parsel; // so that people can experiment in the console
 
+const url = new URL(location);
+if(url.searchParams.get("selector")) {
+    let selector = url.searchParams.get("selector");
+    selectorText.value = selector || selectorText.value; // set default value if the selector is empty
+    url.searchParams.set("selector", selectorText.value);
+}
+
 (tryout.oninput = e => {
-	tokens.textContent = JSON.stringify(parsel.tokenize(selectorText.value), null, "\t");
-	Prism.highlightElement(tokens);
+        url.searchParams.set("selector", selectorText.value);
+        history.replaceState(null, '', `${location.pathname}?${url.searchParams.toString()}`);
+            
+        tokens.textContent = JSON.stringify(parsel.tokenize(selectorText.value), null, "\t");
+        Prism.highlightElement(tokens);
```

Comment:
It appears many smart tabs have been converted to spaces. Could you please revert the whitespace changes?